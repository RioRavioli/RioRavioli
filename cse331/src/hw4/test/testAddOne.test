# Tests for empty graph.
# Tests adding only one node.

CreateGraph graph
ListNodes graph
AddNode graph node
ListNodes graph
