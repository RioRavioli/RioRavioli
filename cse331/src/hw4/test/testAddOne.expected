# Tests for empty graph.
# Tests adding only one node.

created graph graph
graph contains:
added node node to graph
graph contains: node
